types/versionlockstring.pp

```diff
@@ -3,30 +3,30 @@
 # `%{EPOCH}:%{NAME}-%{VERSION}-%{RELEASE}.%{ARCH}`.  As a Regex, it
```

Comment:
This type can use a `@summary` since the summary doesn't render. Was already the case, but maybe good to fix while you're at it.